src/Symfony/Component/Form/Extension/Core/Type/MoneyType.php

```diff
@@ -84,13 +86,23 @@ public function configureOptions(OptionsResolver $resolver)
 
         $resolver->setAllowedTypes('html5', 'bool');
 
+        $resolver->setAllowedValues('model_type', ['float', 'integer']);
+
         $resolver->setNormalizer('grouping', static function (Options $options, $value) {
             if ($value && $options['html5']) {
                 throw new LogicException('Cannot use the "grouping" option when the "html5" option is enabled.');
             }
 
             return $value;
         });
+
+        $resolver->setNormalizer('divisor', static function (Options $options, $value) {
```

Comment:
A minor nitpick: it would make more sense to target `model_type` because it's the option being validated here, and it aligns better with the exception message.

Reply:
Yeah, you're right. Changed.